doc/source/math/direction.rst

```diff
@@ -185,6 +185,13 @@ Structure
 
     :struct:`Vector` of length 1 that is in the same direction as the "starboard side" of this Direction.  Note that it is the same meaning as "what the X axis of the universe would be rotated to if this rotation was applied to the basis axes of the universe". When you LOCK STEERING to a direction, that direction's STARVECTOR is the vector the right wing of the ship will orient to.  SHIP:FACING:STARVECTOR is the way the ship's right wing is aimed right now.
 
+.. attribute:: Direction:INVERSE
+
+    :type: :struct:`Direction`
+    :access: Get only
+    
+    :struct: Gives a `Direction` with the opposite rotation around its axes. So `r(30,15,5)` becomes `r(-30,-15,-5)`.
+    
```

Comment:
I didn't test whether Rots ever get printed as negative but i did test it accepting those rotations. Have just done more testing on them and using `vang` there is a 0.5deg (might be floating point error - might not) between `r(30,15,5):inverse:vector` and `r(-30,-15,5):vector`. There is also a similar difference between them and `r(330,345,355):vector`.
Also `print r(30,15,5):inverse. gives `R(329.756,345.538,2.968)` the first 2 are close enough that i just assumed floating point but the `2.968` makes me uncertain.